packages/passport-interface/src/SubscriptionManager.ts

```diff
@@ -154,6 +171,32 @@ export class FeedSubscriptionManager {
     return responses;
   }
 
+  private async generateCredential(
```

Comment:
I think this logic should live outside of subscription manager. i think if we do it this way, we're overfitting the subscription manager to the semaphore signature pcd use-case. ppl should be able to use this stuff without ever having to think about the particulars of a specific PCD.

Reply:
Another thing that would be good is to cache these credentials, so we don't have to re-generate the credential every time - it's an expensive operation

Reply:
This is now done inside of credential manager!